iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensatorAdder.java

```diff
@@ -32,6 +32,22 @@
 
     ShuntCompensatorAdder setCurrentSectionCount(int currentSectionCount);
 
+    default ShuntCompensatorAdder setRegulatingTerminal(Terminal regulatingTerminal) {
+        throw new UnsupportedOperationException();
+    }
+
+    default ShuntCompensatorAdder setVoltageRegulatorOn(boolean voltageRegulatorOn) {
+        throw new UnsupportedOperationException();
+    }
+
+    default ShuntCompensatorAdder setTargetV(double targetV) {
+        throw new UnsupportedOperationException();
+    }
+
+    default ShuntCompensatorAdder setTargetDeadband(double targetDeadband) {
+        throw new UnsupportedOperationException();
```

Comment:
Here do you throw exceptions because with older versions of IIDM the new attributes are not supported?

Reply:
No, `default` implementation in Java interfaces are just here to prevent breaking changes in implementations of this interface. The exception is just a warning that by default, this method is not implemented.